modules/sdk-coin-bld/test/integration/index.ts

```diff
@@ -0,0 +1,16 @@
+import 'should';
+
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+
+import { Bld, Tbld } from '../../src/index';
+
+describe('BLD', function () {
+  let bitgo: TestBitGoAPI;
+  before(function () {
+    bitgo = TestBitGo.decorate(BitGoAPI, { env: 'mock' });
+    bitgo.safeRegister('bld', Bld.createInstance);
+    bitgo.safeRegister('tbld', Tbld.createInstance);
+    bitgo.initializeTestVars();
+  });
+});
```

Comment:
Do we need this ?

Reply:
We don't have much clarity on integration tests now. So same as all other coins, we are adding this here, but in future, we need some better strategy to have better integration testcases.